app/assets/stylesheets/components/_organizations.scss

```diff
@@ -0,0 +1,63 @@
+.organization-details {
+  h2, h3, h4, h5 {
+    margin: 0;
```

Comment:
Properties should be ordered font, margin<br>Each selector in a comma sequence should be on its own line